protocols/ipfs.ts

```diff
@@ -55,16 +75,46 @@ export class IPFSProtocol implements Protocol<Static<typeof IPFSProtocolFields>>
     const datastore = new FsDatastore(datastorePath)
     const blockstore = new FsBlockstore(blockstorePath)
 
+    const tcpPort = await getPort({ port: 7976 })
+    const wsPort = await getPort({ port: 7977 })
+
     // Default libp2p config: https://github.com/ipfs/helia/blob/main/packages/helia/src/utils/libp2p-defaults.ts
     const libp2pOptions = {
       ...libp2pDefaults(),
+      addresses: {
+        listen: [
+          `/ip4/0.0.0.0/tcp/${tcpPort}`,
+          `/ip4/0.0.0.0/tcp/${wsPort}/ws`,
+          `/ip6/::/tcp/${tcpPort}`,
+          `/ip6/::/tcp/${wsPort}/ws`,
+          '/p2p-circuit'
+          // Uncomment below if WebRTC Direct is needed
```

Comment:
Why not have it by default? This should help browser peers and NAT bypassing

Reply:
Yeah, although WebRTC-direct won't help with NAT traversal. In fact, if you are behind NAT and there's no UPnP, `webrtc` is your only hope (https://github.com/libp2p/js-libp2p/tree/main/packages/transport-webrtc/src/private-to-private)

Reply:
If you do have a public IP, AutoTLS and Secure WebSockets will likely work better than WebRTC-direct. But we have both enabled by default.